### PR TITLE
Doc layout & reference updates

### PR DESCRIPTION
**Pull request by dowen12**
*Issue #, if available:*

*Description of changes:*
New content & updating specific to partner_editable

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: deployment_steps.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-:specificdir: specific
+:specificdir: partner_editable
```
Comment (andrew-glenn) [inline deployment_steps.adoc:1]: Remove this please, as I just tested that inheritence from `_layout_cfn.adoc` works.

File: overview.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-:specificdir: specific
+:specificdir: partner_editable
```
Comment (andrew-glenn) [inline overview.adoc:1]: see above

File: planning_deployment.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 
-:specificdir: specific
+:specificdir: partner_editable
```
Comment (andrew-glenn) [inline planning_deployment.adoc:2]: see above

File: deployment_steps.adoc (hunk 1 above)
Comment (dowen12) [reply deployment_steps.adoc]: Removed references - see latest commit

File: overview.adoc (hunk 2 above)
Comment (dowen12) [reply overview.adoc]: see above

File: planning_deployment.adoc (hunk 3 above)
Comment (dowen12) [reply planning_deployment.adoc]: see above

**Comment by andrew-glenn** [comment]
negated via out of band commits.

**Pull request closed without merging**